upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -510,6 +510,102 @@ There you can find links to upgrade notes for other versions too.
         +   ): ProductFilterCountData;
         ```
 
+- add cart detail on hover ([#1565](https://github.com/shopsys/shopsys/pull/1565))
+  
+  - you can skip this task if you have your custom design
+  - Add new file [src/Resources/styles/front/common/layout/header/cart-detail.less](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Resources/styles/front/common/layout/header/cart-detail.less)
```

Comment:
can you please put all file paths inside the code block (backtick)?
 So it's unified with the rest of the upgrade file.